source/unsafe-messenger.js

```diff
@@ -73,6 +74,34 @@ export default function unsafeMessenger() {
 			editor.blur();
 		});
 	}
+
+	function monacoEditor(target) {
+		const data_uri = target.getAttribute("data-uri");
+		const editor = monaco.editor.getModels(data_uri)[0];
+		let currentValue = editor.getValue();
+		sendBack(target, currentValue);
+		const throttledSend = throttle(50, sendBack);
+
+		editor.onDidChangeContent((e) => {
+			if (e.isUserChange) {
+				if (currentValue !== editor.getValue()) {
+					throttledSend(target, editor.getValue());
```

Comment:
I think this isn't working from the browser to the editor

![ko](https://user-images.githubusercontent.com/1402241/215008299-15610ca1-b385-475b-95b0-7607e67e9fb3.gif)